cluster/addons/kube-proxy/kube-proxy-ds.yaml

```diff
@@ -0,0 +1,87 @@
+# Please keep kube-proxy configuration in-sync with:
+# cluster/saltbase/salt/kube-proxy/kube-proxy.manifest
+
+apiVersion: extensions/v1beta1
```

Comment:
Just realized we have removed support for debian masters in GCE: https://github.com/kubernetes/kubernetes/pull/41666. So I removed some unnecessary salt configuration from this addon file. cc @bowei